website/content/en/docs/setup/installation/package-managers/helm.md

```diff
@@ -49,32 +53,110 @@ sinks:
 VALUES
 ```
 
-Install Vector:
+### Installing
+
+Once you add the Vector Helm repo, and added a Vector configuration file, install the Vector Agent:
+
+```shell
+helm install vector timberio/vector-agent \
+  --namespace vector \
+  --create-namespace \
+  --values values.yaml
+```
+
+### Updating
+
+Or to update the Vector Agent:
+
+```shell
+helm repo update && \
+helm upgrade vector timberio/vector-agent \
+  --namespace vector \
+  --reuse-values
+```
+
+## Aggregator
+
+The Vector [Aggregator] lets you [transform] your data. For example, dedupe, aggregate, or redact the data before sending it to its final destination.
```

Comment:
I might copy the aggregator description @spencergilbert has over here: https://github.com/timberio/vector/pull/8800/files#diff-8a968d252761d4b71779d1117a16b224977b3bad224511172b871192f86617adR26-R33 . I think that one has a bit more color.

Reply:
I was trying to keep it more minimal than the big description (though I do agree that one is good). Would this be a good middle ground:

```
The Vector [Aggregator] lets you [transform] and ship data collected by other agents. For example, it can insure that the data you are collecting is scrubbed of sensitive information, properly formatted for downstream consumers, sampled to reduce volume, and more.
```

Reply:
👍 good point, it does sense to keep it more concise on this page. I like your suggestion.